truss/cli/cli.py

```diff
@@ -377,9 +423,45 @@ def push(
         tr.spec.config.write_to_yaml_file(tr.spec.config_path, verbose=False)
 
     # TODO(Abu): This needs to be refactored to be more generic
-    _ = remote_provider.push(tr, model_name, publish=publish, trusted=trusted)  # type: ignore
+    service = remote_provider.push(tr, model_name, publish=publish, trusted=trusted)  # type: ignore
+
+    model_overview_url = service.resource_url
+    click.echo(f"✨ Model {model_name} was successfully pushed ✨")
 
-    click.echo(f"Model {model_name} was successfully pushed.")
+    if not publish:
+        draft_model_text = """
+|---------------------------------------------------------------------------------------|
```

Comment:
adding more information after `truss push`. we can iterate on the copy here.

Reply:
should we offer to open this page like we do with the `truss watch`?

Reply:
Sure -- updated